app/routes/_gcn.circulars.edit.$circularId/CircularEditForm.tsx

```diff
@@ -98,11 +102,13 @@ function SyntaxReference() {
 
 export function CircularEditForm({
   formattedContributor,
+  circular,
   defaultBody,
   defaultSubject,
   searchString,
 }: {
   formattedContributor: string
+  circular?: Circular
   defaultBody: string
   defaultSubject: string
```

Comment:
It seems odd that this takes  both the complete Circular _and_ the default subject and body.

Reply:
Yea good point, I can refactor this a bit better